ee/clickhouse/process_event.py

```diff
@@ -124,16 +126,10 @@ def _capture_ee(
 
 
 def check_and_create_person(team_id: int, distinct_id: str) -> Optional[Person]:
-    person = get_person_by_distinct_id(team_id=team_id, distinct_id=distinct_id)
+    person = Person.objects.get(team_id=team_id, persondistinctid__distinct_id=distinct_id)
     if person:
         return person
```

Comment:
There is a race condition here where the normal capture path may have created this user (and they were in fact new) and we miss it, never emitting the new user to kafka. We will need to sync somehow. Maybe by adding a new field on Person for emitted to kafka/CH?